addons/project/security/project_security.xml

```diff
@@ -146,7 +146,7 @@
         <field name="name">Project: See private tasks</field>
         <field name="model_id" ref="project.model_project_task"/>
         <field name="domain_force">[
-            ('project_id.privacy_visibility', '!=', 'followers'),
+            '|', ('project_id', '=', False), ('project_id.privacy_visibility', '!=', 'followers'),
```

Comment:
the following part of the domain is:
```
                  '|',
                      ('project_id', '!=', False),
                      ('parent_id', '!=', False),
```
which sounds correct to me. Shouldn't we just put it first? Not sure I understand actually :thinking:

Reply:
> the following part of the domain is:
> 
> ```
>                   '|',
>                       ('project_id', '!=', False),
>                       ('parent_id', '!=', False),
> ```
> 
> which sounds correct to me. Shouldn't we just put it first? Not sure I understand actually thinking

I think that this task is not relevant anymore since the changes introduced in https://github.com/odoo/odoo/pull/128281

Reply:
Isn't it needed in 16.3?

Reply:
> Isn't it needed in 16.3?

We cannot change the record rules on existing db :( That's why I did the change in master at that time. Sorry for the late reply btw